packages/core/helper-plugin/src/components/ContentBox.tsx

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
-
-import { Flex, Typography } from '@strapi/design-system';
-import PropTypes from 'prop-types';
+import { Flex, FlexProps, Typography } from '@strapi/design-system';
 import styled from 'styled-components';
 
+export interface ContextBoxProps {
+  title?: string;
+  subtitle?: string;
+  icon?: FlexProps['children'];
+  iconBackground?: FlexProps['background'];
+  endAction?: FlexProps['children'];
+  titleEllipsis?: boolean;
+}
```

Comment:
Actually couldn't extend and Pick a Partial, similar to the suggestion here: https://github.com/strapi/strapi/pull/17732#discussion_r1301104847